core/src/main/java/org/elasticsearch/bootstrap/BootstrapCheck.java

```diff
@@ -224,12 +227,25 @@ boolean isMemoryLocked() {
 
     }
 
+    static class MinMasterNodesCheck implements Check {
+
+        @Override
+        public boolean check(Settings settings) {
+            return settings.get("discovery.zen.minimum_master_nodes") == null;
+        }
+
+        @Override
+        public String errorMessage() {
+            return "please set [discovery.zen.minimum_master_nodes] to a majority of the number of master eligible nodes in your cluster.";
```

Comment:
Can you replace the string `discovery.zen.minimum_master_nodes` with `ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTINGS.getKey()`?